addons/event_crm/views/event_event_views.xml

```diff
@@ -4,6 +4,7 @@
         <field name="name">event.event.form.inherit.event.crm</field>
         <field name="model">event.event</field>
         <field name="inherit_id" ref="event.view_event_form"/>
+        <field name="priority" eval="9"/>
```

Comment:
Typo in your commit message
> co**mmm**it instead of commit

You should also add the link to the enterprise pull request just after the Task ID